saleor/graphql/product/sorters.py

```diff
@@ -139,25 +140,30 @@ class ProductOrderField(graphene.Enum):
     TYPE = ["product_type__name", "name", "slug"]
     PUBLISHED = ["is_published", "name", "slug"]
     PUBLICATION_DATE = ["publication_date", "name", "slug"]
+    COLLECTION = ["row_number"]
     RATING = ["rating", "name", "slug"]
 
     @property
     def description(self):
         # pylint: disable=no-member
         descriptions = {
-            ProductOrderField.NAME.name: "name",
-            ProductOrderField.PRICE.name: "price",
-            ProductOrderField.TYPE.name: "type",
+            ProductOrderField.COLLECTION.name: (
+                "collection. Note: "
```

Comment:
What will happen if I will use it with a different query than the one from  the note?

Reply:
You will get sorted products but the behavior of the sorter will be different than you expect because you're using wrong query.